Docs stacker retrofit sn revision

jwwojak opened the pull request:
# Overview

Text for the Stacker upgrade kit is causing some confusion in the field. This change strengthens language in the Flex and Stacker manuals that indicates the upgrade kit is _required_ if a Flex has A10 or A20 in the serial number, with or without the HEPA/UV module.

Robots with **A30** in their serial numbers do not require this upgrade.

**Sandbox:**
- Stacker manual: https://sandbox.docs.opentrons.com/docs-stacker-retrofit-sn-revision/stacker/compliance/#flex-stacker-and-hepauv-compatibility

- Flex manual: https://sandbox.docs.opentrons.com/docs-stacker-retrofit-sn-revision/flex/modules/stacker/#module-compatibility

RTC-893

## Test Plan and Hands on Testing

Read the wordz for clarity and accuracy.

## Changelog

Revises text here:

- `flex/docs/modules/stacker.md#module-compatibility`
- `stacker/docs/compliance#flex-stacker-and-hepa/uv-compatibility`

This change will also update the [retrofit kit guide](https://docs.google.com/document/d/1UxfxRWObhTvgp34Y_7ZcCkdmS3bi65ohzewiHTC0QjY/edit?usp=sharing) with the more comprehensive text used by the Stacker manual after this PR is reviewed and approved.

## Review requests

Does this change make it clear that the retrofit kit is required for older model robots, even without the HEPA module?

## Risk assessment

Medium: can cause field service confusion.

jwwojak committed:
Revise Stacker retrofit text and requirements

Make the docs more clear that the kit is required on some model robots, even if the customer does not have a HEPA/UV module.

jwwojak committed:
Changes for the Flex manual, modules section

Shorter than the Stacker manual but links to that resource with the more comprehensive explanation.

ecormany commented on docs/flex/docs/modules/stacker.md:
```diff
@@ -57,9 +57,10 @@ The Stacker accepts Opentrons Flex tip racks, selected items in our [Labware Lib
+- Allows early model robots to recognize and communicate with the Staker.
+- Prevents the HEPA/UV module from operating if the Stacker is improperly installed.
 
 For additional information, see the [HEPA/UV compatibility section](../../stacker/compliance.md#stacker-and-hepauv-compatibility) of the Stacker instruction manual. You can also contact [Opentrons Sales](https://opentrons.com/contact) if you're unsure about a robot's manufacture date and/or have a model that needs to be upgraded.
```
This anchor needs to change, since you updated the header there.
```suggestion
For additional information, see the [HEPA/UV compatibility section](../../stacker/compliance.md#flex-stacker-and-hepauv-compatibility) of the Stacker instruction manual. You can also contact [Opentrons Sales](https://opentrons.com/contact) if you're unsure about a robot's manufacture date and/or have a model that needs to be upgraded.
```

ecormany commented on docs/stacker/docs/compliance.md:
```diff
@@ -150,17 +150,26 @@ You can install and use Stackers with the [Opentrons Flex HEPA/UV Module](https:
+
+Compare these serial number examples to the serial number on your robot to determine if it needs the Stacker retrofit kit installed.
+
+| Serial number identifier | Example | Retrofit required |
```
If you want a shorter header over the first column, you could go with "Version". That's what we call it in the part of the manual that explains the [serial number format](https://sandbox.docs.opentrons.com/docs-stacker-retrofit-sn-revision/flex/system-description/specs/#serial-number).

So if you go that way: `| Version | Serial number example |  Retrofit required |`

jwwojak replied:
Thanks! I wasn't sure what to call it.

ecormany reviewed (approved):
This looks really great. Definitely lays out the situation more clearly.

One link to update (it still goes to the right page, but not the intended section) and one suggestion. But otherwise :shipit:

jwwojak committed:
Apply suggestions from code review

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
Making reviewer changes

Adding text that includes the correct name of this part of the serial number, the "version."

Linking to Flex manual section on serial numbers too.
